sanghyeok/LV_1/핸드폰-번호-가리기.js

```diff
@@ -0,0 +1,4 @@
+function solution(phone_number) {
+    let star = phone_number.length - 4;
+    return "*".repeat(star) + phone_number.substring(star);
+}
```

Comment:
```suggestion
function solution(phone_number) {
    let star = phone_number.length - 4;
    return "*".repeat(star) + phone_number.substring(star);
}
```
substring을 활용한 풀이 잘 봤습니다!